prow/plugins/bugzilla/bugzilla.go

```diff
@@ -337,7 +337,21 @@ Once a valid bug is referenced in the title of this pull request, request a bug
 				e.bugId, bc.Endpoint()))
 		}
 
-		valid, why := validateBug(*bug, options)
+		var dependents []bugzilla.Bug
+		if options.DependentBugStatuses != nil {
+			for _, id := range bug.DependsOn {
+				dependent, err := bc.GetBug(id)
```

Comment:
We can fetch all dependent items with one request:

https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#get-bug

> You can also use Search Bugs to return more than one bug at a time by specifying bug IDs as the search terms.
> `GET /rest/bug?id=12434,43421`

And potentially get only the fields we care about:

https://bugzilla.readthedocs.io/en/latest/api/core/v1/general.html#useful-parameters

> `include_fields`: The (case-sensitive) names of fields in the response data. Only the fields specified in the object will be returned, the rest will not be included. Fields should be comma delimited.

But beware of Bugzilla's amazing API:

> Invalid field names are ignored.

Reply:
Using one request could be nice but It seemed like premature optimization -- harder to implement than just changing our current method since it uses parameters instead of path ...